src/files/normalise-input.js

```diff
@@ -0,0 +1,233 @@
+'use strict'
+
+const errCode = require('err-code')
+const { Buffer } = require('buffer')
+
+/*
+ * Transform one of:
+ *
+ * ```
+ * Buffer|ArrayBuffer|TypedArray
```

Comment:
For the sake of simplicity and understanding maybe we should setup an alias for this:

```suggestion
 * Bytes (Buffer|ArrayBuffer|TypedArray|Iterable<Number>)
 * Bloby (Blob|File)
```

When I see `Bytes` I think that they are part of a set of bytes for a single file (albeit there may only be one), when I see `Bloby` I think that this is a whole file.